docs/content/en/rules/untrusted_checkout_exec.md

```diff
@@ -194,8 +194,61 @@ jobs:
           })
 ```
 
+### Azure DevOps
+
+#### Caveat
+False positives are likely given that static analysis of solely the pipeline file is not enough to confirm exploitability
+
+#### Recommended
+##### Azure DevOps Settings
+Organization Setting:
+![img.png](img.png)
+
+Avoid activating the following settings to prevent issues:
+![img_1.png](img_1.png)
+
+```yaml
+pr:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  - name: someVariable
+    value: someValue
+
+steps:
+  - checkout: self
+    path: trusted
+  - script: npm ci
+    workingDirectory: trusted
+
+```
```

Comment:
I think it's better to remove the code example, just keep the screenshots for now.

To do the "trusted" tool setup we'd really need to specify a trusted ref (like explained here https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checking-out-a-specific-ref)

GitHub Actions and ADO syntax is quite different and I don't have time to study in depth the best practice right now.

Let's just ship with telling people to disable and we'll see how to explain a way to do that safely later....